core/src/utils/keyboard/keyboard-controller.ts

```diff
@@ -0,0 +1,46 @@
+import { win } from '../window';
+
+/**
+ * Creates a controller that tracks and reacts to opening or closing the keyboard.
+ *
+ * @internal
+ * @param keyboardChangeCallback A function to call when the keyboard opens or closes.
+ */
+export const createKeyboardController = (keyboardChangeCallback?: (keyboardOpen: boolean) => void) => {
```

Comment:
Can we update the return type of this function so that it uses the custom `KeyboardController` type?